sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -781,12 +789,15 @@ public Mono<Void> openConnectionsAndInitCaches(int numProactiveConnectionRegions
                     .setProactiveConnectionRegionsCount(numProactiveConnectionRegions)
                     .build();
 
-            return withContext(context -> openConnectionsAndInitCachesInternal(proactiveContainerInitConfig)
-                    .flatMap(
-                        openResult -> {
-                            logger.info("OpenConnectionsAndInitCaches: {}", openResult);
-                            return Mono.empty();
-                        }));
+            return withContext(context -> openConnectionsAndInitCachesInternal(
+                    proactiveContainerInitConfig
+            )
+                .collectList() // TODO: need a way to get open result
+                .flatMap(
+                    openResult -> {
+                        logger.info("OpenConnectionsAndInitCaches: {}", openResult);
```

Comment:
Does this really have to be info level? vs. debug or even trace?

Reply:
Not necessarily - can be debug / trace.